validator/testdata/feature_tests/amp-yotpo.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
This and the out file are missing the license at the top.

Reply:
added